src/couchapps/ReqMgr/updates/updaterequest.js

```diff
@@ -36,6 +36,19 @@ function(doc, req) {
     	return true;
 	}
 	
+	function updateTaskStepChain(chainType, prop, value) {
+		if (doc[chainType + "Chain"]) {
+			var numChain = doc[chainType + "Chain"];
```

Comment:
BTW, this whole function is miss-indented. And so is L88-90. Maybe you can just reformat the whole file to make it more readable :)